content/docs/other-guides/multi-user-overview.md

```diff
@@ -151,6 +151,32 @@ The following resources are created as part of the profile creation:
 Namespaces, Profiles and Namespaces are sometimes used interchangably in the
 documentation.
 
+### Cloud API Authenticate with Profile
+
+**GCP**: User profile on GCP will be integrated with [GKE Workload Identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity).
+User pods will be authenticated automatically when talk to GCP APIs.
+
+By default all profiles in a cluster will share a public GCP service account identity.
```

Comment:
Could we clarify what is meant by default here?
I assume this is only true if user deployed with kfctl_gcp_iap.yaml or if their KFDef contains the GCPPluginSpec.

So we could say something like that e..g

"If you deployed Kubeflow following the GCP instructions then the profiler controller will automatically bind the "default-editor" service account to the GCP service account".